src/ReactDataGrid.js

```diff
@@ -857,7 +857,7 @@ const ReactDataGrid = React.createClass({
   },
 
   dragEnabled: function(): boolean {
-    return this.props.onGridRowsUpdated !== undefined || this.props.onCellsDragged !== undefined;
+    return (this.props.onGridRowsUpdated !== undefined || this.props.onCellsDragged !== undefined) && this.props.enableDragAndDrop !== false;
```

Comment:
Hi @diogofcunha,

The down side of your proposal would be that if enableDragAnDrop is not explicitly set to true (ie. undefined) DragAndDrop is suddenly disabled. It does make sense though to put the enable test at the beginning though.

And would it make sense to turn of the "drag" box when DragAndDrop is disabled (at the moment it isn't)

Please advice.

Thomas